app/src/main/java/org/oppia/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -73,8 +75,13 @@ class ExplorationActivityPresenter @Inject constructor(
     }
 
     explorationToolbar = binding.explorationToolbar
+    explorationToolbarTitle = binding.explorationToolbarTitle
     activity.setSupportActionBar(explorationToolbar)
 
+    binding.explorationToolbar.setOnClickListener {
+      binding.explorationToolbarTitle.isSelected = true
```

Comment:
Should we start moving text only if user click on it? 

Based on this implementation, text-only start moving is a user click on it which I think is not a good idea. Ideally, the text should move when it's long, no other parameters for moving thing.

Reply:
This was actually suggested by Chantel and the reason why this is correct is that, if we keep it moving by default, it will draws user attention which is actually not needed in this toolbar title. Also, because user must have seen the title in full length already on previous screen we should not draw user's attention it.
For example, on home screen, user can see full topic name and once they got inside there also they can see topic name in `info` tab so we should not bring his/her focus on toolbar. Similar thing happens for other items also

Reply:
This make sense. Thanks.